src/v2/Components/Artwork/Details.tsx

```diff
@@ -294,6 +310,7 @@ export const DetailsFragmentContainer = createFragmentContainer(Details, {
       href
       title
       date
+      is_saved
```

Comment:
**relay-question:** Do we need to include `is_saved` here to make it accessible in this component? Asking cause down below we are spreading the `NewSaveButton_artwork` which made me thing that since it includes the `is_saved` it would make it accessible in the component but it didn't 🤔

Reply:
if `is_saved` should be used in `Details` component (as far as I understand, we should display the "save" button without a hover effect if the user saved artwork earlier), then you need to query it here

Reply:
> Asking cause down below we are spreading the NewSaveButton_artwork which made me thing that since it includes the is_saved it would make it accessible in the component but it didn't

in `Details` component, only the data that you explicitly requested will be available (data from fragments will not be available). You can check the available data in an automatically [generated GraphQL file](https://github.com/artsy/force/blob/e03d6679830be82c9458bdbda77fe23b7535631d/src/v2/__generated__/Details_artwork.graphql.ts#L7)